scripts/sort_sparse_matrix.py

```diff
@@ -0,0 +1,53 @@
+"""Python Function to sort sparse expression matrix files
+Arguments:
```

Comment:
Please remove this manual "Arguments" documentation and use ArgParse's built-in `--help` functionality, which pulls content from `add_argument` (e.g. [here](https://github.com/broadinstitute/single_cell_portal/pull/30/files#diff-f7dd3f929225170235d70eff11c82c8fR46)).  

The current approach here of having such content in two places will increase the chance of documentation diverging over time, or (unnecessarily) having to remember to update it in two places upon every change.

Reply:
So should there be any comments at the top of the file?

Reply:
I think a single comment saying what the file does is sufficient.  Eric is correct in that the --help line will output information about arguments, and I think people using the script will look there for info.

Reply:
Yes; although the argument documentation should be removed, we'll want to keep a file-level docstring like you have here.  (Perhaps remove "Python Function to" -- the users presumably knows they're dealing with Python here, and "Function to" doesn't add much.)  I also suggest adding at least one example call.

Try using `RawDescriptionHelpFormatter` in your ArgParse constructor, as done [here](https://github.com/broadinstitute/single_cell_portal/blob/ade3d47e6aebb2d0aa54bce5f0779989a291905f/scripts/test_files/big_study/make_toy_data.py#L29).  That will ensure your file docstring and any examples are included in `--help` output, beside your documentation from `add_argument`.